packages/ruleset/src/rules/no-ambiguous-paths.js

```diff
@@ -0,0 +1,19 @@
+/**
+ * Copyright 2017 - 2023 IBM Corporation.
```

Comment:
I have a question - are these meant to reflect the life of the project or the individual file? I haven't paying attention to that in reviews but in my API symmetry branch I've been using just `Copyright 2023 IBM Corporation.`. Wondering if I should go update mine 🙂

Reply:
Technically, when you introduce a new file the initial copyright statement should include only the year that the file was introduced, then later an additional year added as needed when/if the file is updated. I've just been really lazy when adding new rules, as I just copy an existing file.  I don't think the copyright police will care all that much, but technically we should only include "2023" in these.  I'll change 'em.

Reply:
fixed